docs/swagger_v2/activities.spec.yaml

```diff
@@ -57,7 +57,6 @@ schemas:
           - reward_dev
           - reward_block
           - fee_refund_oracle
-          - fee_lock_name
```

Comment:
noticed this is different from the `type: string` error, is it a wanted change?

Reply:
It's not, reverted it to include it on separate PR. Thanks